corehq/sql_accessors/sql_templates/get_case_last_modified_dates_by_range.sql

```diff
@@ -0,0 +1,33 @@
+DROP FUNCTION IF EXISTS get_case_last_modified_dates_by_range(TEXT, TIMESTAMP WITH TIME ZONE, TIMESTAMP WITH TIME ZONE);
+
+CREATE FUNCTION get_case_last_modified_dates_by_range(
```

Comment:
having these functions isn't strictly necessary unless you require that the queries get run in parallel on all DBs. For this use case I would consider doing the query one shard at a time directly from python:
```python
for db in get_db_aliases_for_partitioned_query():
    yield from CommCareCaseSQL.objects.using(db).filter(...).values_list('case_id')
```
```

Reply:
Oh, nice, thanks. will update.

Reply:
thanks for the pointer that was much easier.  https://github.com/dimagi/commcare-hq/pull/25479/commits/cd5fe3a180a7c55cfd80a5249f98f3a6f7916be7